cmd/localkube/cmd/options.go

```diff
@@ -59,6 +60,8 @@ func AddFlags(s *localkube.LocalkubeServer) {
 	flag.IntVar(&s.APIServerPort, "apiserver-port", s.APIServerPort, "The port the apiserver will listen securely on")
 	flag.IPVar(&s.APIServerInsecureAddress, "apiserver-insecure-address", s.APIServerInsecureAddress, "The address the apiserver will listen insecurely on")
 	flag.IntVar(&s.APIServerInsecurePort, "apiserver-insecure-port", s.APIServerInsecurePort, "The port the apiserver will listen insecurely on")
+	flag.StringVar(&s.APIServerName, "apiserver-name", s.APIServerName, "The apiserver name for localkube/kubernetes")
```

Comment:
Can we please add something to the effect that this name is used in the generated certificate? Will be helpful to someone just running help commands.